### PR TITLE
Fix: AT-X

### LINKED ISSUE #3273: Fix: AT-X
You Need to change AT-X streaming location because when I watch the AT-X on media player I'm using it says 
"Your input can't be opened 
VLC is unable to open the MRL 'https://sub2.neetball.net/live/neet.m3u8'."
i try to open neetball.net when i click link the first link i need to login first.

**_Channel Name:_** AT-X
**_Broken Link (from playlist):_** https://sub2.neetball.net/live/neet.m3u8
**_Possible Replacement (optional):_** https://sub2.neetball.net/live/cd1212.m3u8
**_Notes (optional):_** ~

### PR DESCRIPTION
**Pull request by Dum4G**
Closes https://github.com/iptv-org/iptv/issues/3273

### CODE REVIEW

**Comment by LaneSh4d0w** [comment]
CD1212 doesn't seem to work for now.

File: channels/jp.m3u [hunk 1]
```diff
@@ -1,13 +1,15 @@
...
 http://210.210.155.35/dr9445/h/h02/01.m3u8
-#EXTINF:-1 tvg-id="AnimaxJapan.jp" tvg-name="Animax Japan" tvg-country="JP" tvg-language="Japanese;Indonesian" tvg-logo="" group-title="",Animax Japan (Indonesian Subs)
+#EXTINF:-1 tvg-id="AnimaxJapan.jp" tvg-name="Animax Japan" tvg-country="JP" tvg-language="Japanese;Indonesian" tvg-logo="" group-title="Animation",Animax Japan (Indonesian Subs)
 http://210.210.155.35/dr9445/h/h144/01.m3u8
-#EXTINF:-1 tvg-id="AnimaxJapan.jp" tvg-name="Animax Japan" tvg-country="JP" tvg-language="Japanese;Vietnamese" tvg-logo="" group-title="",Animax Japan (Vietnamese Subs)
+#EXTINF:-1 tvg-id="AnimaxJapan.jp" tvg-name="Animax Japan" tvg-country="JP" tvg-language="Japanese;Vietnamese" tvg-logo="" group-title="Animation",Animax Japan (Vietnamese Subs)
 https://livecdn.fptplay.net/hda3/animaxport_2000.stream/.m3u8
-#EXTINF:-1 tvg-id="ATX.jp" tvg-name="AT-X" tvg-country="JP" tvg-language="Japanese" tvg-logo="" group-title="",AT-X
+#EXTINF:-1 tvg-id="ATX.jp" tvg-name="AT-X" tvg-country="JP" tvg-language="Japanese" tvg-logo="https://www.at-x.com/favicon_android.png" group-title="Animation",AT-X
+https://sub2.neetball.net/live/cd1212.m3u8
```
Comment (freearhey) [inline channels/jp.m3u:11]: image

**Review by freearhey — CHANGES_REQUESTED** [review]

File: channels/jp.m3u (hunk 1 above)
Comment (MapGuy11) [reply channels/jp.m3u:11]: image
404 Not Found

Comment (LaneSh4d0w) [reply channels/jp.m3u:11]: Correct link is : https://sub.neetball.net/live/cd1212.m3u8

Comment (MapGuy11) [reply channels/jp.m3u:11]: image
That Works!

**Review by freearhey — APPROVED** [review]

**Comment by Dum4G** [comment]
image
Reverting unless they figure things out?

**Comment by LaneSh4d0w** [comment]
Due to the fact it changes all the time... it will be difficult to make sure what's in it.

**Pull request merged**